### DISCUSSION TITLE
更新至 v3.25.1 版本后，访问时提示”获取设置失败： 请稍候，正在加载储存“

### DISCUSSION THREAD

**Discussion by KHwang9883** [AlistGo/alist#4953] [Q&A] [answered]
尝试降级、备份数据并重装没有解决问题。

管理后台无法进入，同样提示”获取设置失败： 请稍候，正在加载储存“。

**Comment by Troray** [comment]
查看日志

**Reply by KHwang9883** [reply]
[log.log](https://github.com/alist-org/alist/files/12283214/log.log)
看着没啥有用的信息

**Comment by KHwang9883** [comment]
我全新安装了 alist，添加了 123pan 存储后重启 alist，必然复现此问题

**Comment by louiejordan** [comment]
我也是这个问题，过去更新完几秒钟就能正常进入，现在20分钟了还是进不去

**Comment by sxjeru** [comment, marked as answer]
相同问题。暂时可以去数据库手动去掉123云盘以进入web界面。

**Reply by louiejordan** [reply]
确实是123盘的问题，我的今天可以进入了，后台发现123盘报错，这个网盘太折腾，出好几次问题了
